### PR TITLE
Adding new method for Table.py

### PR DESCRIPTION
**Pull request by yizerozhuang**
Operations for Booking Challenge

 read_csv: read csv file and convert into FeatureTable

 union: append more date row-wisely
```python
# Example for Feature table union method 
# Find the union of two tables according to their columns' name
# :param tbl: feature table
# :return: Feature table
df1.show()
>>> +---+---+
    | x | y |
    +---+---+
    | 1 | a |  
    | 2 | b |
    +---+---+
df2.show()
>>> +---+---+
    | y | x |
    +---+---+
    | c | 3 |  
    | d | 4 |
    +---+---+
df3 = df1.union(df2)
df3.show()
>>> +---+---+
    | x | y |
    +---+---+
    | 1 | a |  
    | 2 | b |
    | 3 | c |
    | 4 | d |
    +---+---+
```
 append_columns: append a new columns with constant value
```python
# Append the columns with value to table
# :param col: the name of the col
# :param value: value to be append
df.show()
>>> +---+---+
    | x | y |
    +---+---+
    | 1 | a |  
    | 2 | b |
    +---+---+
df.append_columns("z",0)
>>> +---+---+---+
    | x | y | z |
    +---+---+---+
    | 1 | a | 0 |
    | 2 | b | 0 |
    +---+---+---+
```

### CODE REVIEW

File: pyzoo/zoo/friesian/feature/table.py [hunk 1]
```diff
@@ -637,6 +650,31 @@ def gen_cat(items):
...
+        :return: Feature Table
+        """
+        if not isinstance(tbls,list):
+            tbls = [tbls]
+        result = self.df
+        for x in tbls:
+            result = result.unionByName(x.df) if withname else result.union(x.df)
+        return FeatureTable(result)
+
+    def append_columns(self,col,value):
```
Comment (hkvision) [inline pyzoo/zoo/friesian/feature/table.py:669]: add space after `,` otherwise will fail for the style check

Comment (yizerozhuang) [reply pyzoo/zoo/friesian/feature/table.py:669]: sure, will do right now

**Comment by hkvision** [comment]
Also need to add unit test here: https://github.com/intel-analytics/analytics-zoo/blob/master/pyzoo/test/zoo/friesian/feature/test_table.py

**Comment by yizerozhuang** [comment]
> Also need to add unit test here: https://github.com/intel-analytics/analytics-zoo/blob/master/pyzoo/test/zoo/friesian/feature/test_table.py

yes I am doing it right now

File: pyzoo/zoo/friesian/feature/table.py [hunk 2]
```diff
@@ -71,15 +79,13 @@ def compute(self):
     def to_spark_df(self):
         """
         Convert the current Table to a Spark DataFrame.
-
```
Comment (hkvision) [inline pyzoo/zoo/friesian/feature/table.py:74]: do not change these...

Comment (yizerozhuang) [reply pyzoo/zoo/friesian/feature/table.py:74]: sorry I will check the current  version and re-commit after

**Pull request closed without merging**